scipy/integrate/_ivp/rk.py

```diff
@@ -495,9 +495,10 @@ def _estimate_error(self, K, h):  # Left for testing purposes.
     def _estimate_error_norm(self, K, h, scale):
         err5 = np.dot(K.T, self.E5) / scale
         err3 = np.dot(K.T, self.E3) / scale
-
         err5_norm_2 = np.sum(err5**2)
         err3_norm_2 = np.sum(err3**2)
+        if err5_norm_2 == 0 and err3_norm_2 == 0:
+            return 0
         denom = err5_norm_2 + 0.01 * err3_norm_2
```

Comment:
In the [reference code](http://www.unige.ch/~hairer/software.html) they do

```fortran
IF (DENO.LE.0.D0) DENO=1.D0
```

But of course the norm should be `>= 0` so I think this amounts to the same thing.

Reply:
Yes, that was my thinking as well.